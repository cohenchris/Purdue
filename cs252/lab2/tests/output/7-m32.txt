0x5759d050
(nil)
[
	addr: 0080
	size: 4000
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

0x5759d098
(nil)
[
	addr: 0152
	size: 3928
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

0x5759d0e0
(nil)
[
	addr: 0224
	size: 3856
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

0x5759d128
(nil)
[
	addr: 0296
	size: 3784
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

