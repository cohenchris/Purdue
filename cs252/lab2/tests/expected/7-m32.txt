
(nil)
[
	addr: 0080
	size: 4000
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]


(nil)
[
	addr: 0152
	size: 3928
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]


(nil)
[
	addr: 0224
	size: 3856
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]


(nil)
[
	addr: 0296
	size: 3784
	left_size: 64
	allocated: false
	prev: NULLPTR
	next: NULLPTR
]

